config/initializers/filter_parameter_logging.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+# Be sure to restart your server when you modify this file.
+
+# Configure sensitive parameters which will be filtered from the log file.
+Rails.application.config.filter_parameters += [:password, :credit_card]
```

Comment:
This list used to be like this in the past but it is much more extense now:
https://github.com/3scale/porta/blob/e2dbb0cf87ed7a340261f681b9444254dd1a199b/config/application.rb#L224-L228

And imo we should define the whole list only once and not twice.